test/drenv/drenv_test.py

```diff
@@ -18,6 +18,21 @@ def test_start(tmpenv):
     assert cluster.status(tmpenv.prefix + "cluster") == cluster.READY
 
 
+def test_dump_unknown():
+    """
+    succeeds as dump shows how drenv binds templates, expands scripts, arguments,
+    name workers and applies default values with respect to .yaml argument
+    even if the cluster is not present
+    """
+    commands.run("drenv", "dump", "--name-prefix", "unknown-", "external.yaml")
```

Comment:
Testing --name-prefix for dump is important, good that you are including it, but we don't need the "unknown-" prefix since dump does not access any clusters. We can use
the same prefix used in the external_test.py.